### ISSUE TITLE
Indeterminate progress bar doesn't seem to work on MacOS

### ISSUE THREAD

**Issue by burtonator** [burtonator/polar-bookshelf#445]

**Comment by jschaf** [comment]
I can confirm on macOS, none of the progress bars show any changes.

**Comment by burtonator** [comment]
you are updating the pagemarks?

**Comment by jschaf** [comment]
Ah, I didn't realize page marks were a manual thing.  It works as expected.

**Comment by burtonator** [comment]
yeah.. the semantics on how they would work automatically are difficult and I need to think them through a bit more.
